scripts/new_version.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+if [ $# != 3 ]; then
+    echo "Usage: $0 <shortVersion> <ubiVersion> <operatorVersion>"
+    echo "Ex: $0 1.90.0 1.90.0-ubi-1 1.90.0-1"
```

Comment:
Q: Can the second two be computed from the first one? Or is this the case where the numbers might change independently if we have to recut them?